README.md

```diff
@@ -0,0 +1,713 @@
+var german = diacritics.getVariant("de");
+/* german =>
+{
+    "de": {
+        "metadata": { ... }
+        "data": { ... }
+    }
+}
+*/
+```
+
+If a variant is not found, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getVariant("test");
+// test => { "message": "Variant 'test' was not found" }
+```
+
+### diacritics.getAlphabet(code)
+
+Returns the metadata and data for all languages matching the given alphabet code.
+
+The `code` parameter (type: `string`) must be set using a [ISO 15924](https://en.wikipedia.org/wiki/ISO_15924) case insensitive script code value, e.g. `Latn`.
+
+The returned value will be an object containing all languages that match the set alphabet.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var latin = diacritics.getAlphabet("latn");
+/* latin =>
+{
+    "de": {
+        "metadata": { ... }
+        "data": { ... }
+    },
+    "es": {
+        "metadata": { ... }
+        "data": { ... }
+    }
+}
+*/
+```
+
+If an alphabet is not found, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getAlphabet("test");
+// test => { "message": "Alphabet 'test' was not found" }
+```
+
+### diacritics.getContinent(code)
+
+Returns the metadata and data for all languages matching the given continent code.
+
+The `code` parameter (type: `string`) must be set using a [ISO-3166](https://en.wikipedia.org/wiki/List_of_sovereign_states_and_dependent_territories_by_continent_%28data_file%29) case insensitive value, e.g. `EU`:
+
+The returned value will be an object containing all languages that match the set continent.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var antarctica = diacritics.getContinent("an");
+/* antarctica =>
+{
+    "es": {
+        "metadata": { ... }
+        "data": { ... }
+    }
+}
+*/
+```
+
+If a continent is not found, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getContinent("test");
+// test => { "message": "Continent 'test' was not found" }
+```
+
+### diacritics.formatUnicode(string)
+
+Converts the escaped unicode as stored within the database (`\\uHHHH` where H = hex) into actual unicode characters. Using `.replace(/\\\u/g, "\u")` will appear to work, but it does not create an actual unicode character; so we must use `.fromCharCode()` to properly convert the string.
+
+The `string` parameter (type: `string`) can contain regular characters as well as the escaped unicode characters. Diacritics within the string will not be modified.
+
+The returned string will have replaced any escaped unicode with the equivalent unicode value
+
+```js
+var diacritics = require("diacritics-transliterator");
+var string = diacritics.formatUnicode("T\\u00E9st");
+// string => "Tést"
+```
```

Comment:
It is needed because the database stores escaped unicode values (e.g. `\\u00E9`). Storing `\u00E9` would be the same as providing the `raw` value.

I'm not sure what you mean by using `String.raw`, but this method converts the double escaped unicode into a raw unicode value.